data/quests.ts

```diff
@@ -1218,4 +1242,583 @@ export const Quests: Array<ListType> = [
+        description: "Reload the area and Boggart will be dying. Dung Eater will invade when you interact with Boggart."
+      },
+      {
+        id: "3ea5714d-943f-44a9-b27b-cb37c8637577",
+        description: "After fighting off Dung Eater, loot Boggart's corpse for a Seedbed Curse, Blackguard's Iron Mask, Iron Ball, and Blackguard's Bell Bearing."
+      }
+    ]
+  },
+  {
+    id: "3c614049-2380-48e2-9727-7dbad5e4e703",
+    name: "Jar-Bairn",
+    url: "https://eldenring.wiki.fextralife.com/Jar+Bairn",
+    requirements: [
+      {
+        id: "fb0351a4-6905-43d2-8b29-0309d8b759b2",
+        description: "Reach Jarburg by following a series of grave stones down the eastern cliffs of Liurnia, just south of the Carian Study Hall.",
+        url: "https://eldenring.wiki.fextralife.com/Jarburg"
+      },
+      {
+        id: "b7e45220-1c65-47ae-a6ca-fa69e017585f",
+        description: "(Optional) Interact with Jar-Bairn in Jarburg for some extra dialogue."
+      },
+      {
+        id: "82bb981b-9034-43ed-a3b7-52681a66fe79",
+        description: "Progress Alexander's quest line at least one step."
+      },
+      {
+        id: "bcab5335-85f7-4cd6-ab98-bcc3ef8fb9db",
+        description: "(Optional?) Progress Diallos' quest line."
+      },
+      {
+        id: "e6870a01-abdf-4f77-a148-5a3121afb358",
+        description: "Exhaust Jar-Bairn's dialogue and reload the area multiple times, until Jar-Bairn mentions poachers in the area."
+      },
+      {
+        id: "03d6abc4-a0f5-4410-8916-81f4d104e4e7",
+        description: "Continue to reload the area and exhaust Jar-Bairn's dialogue."
+      },
+      {
+        id: "2257f951-83a8-42c5-ab23-c028da4847b5",
+        description: "Fully complete Alexander's quest line and obtain Alexander's Innards."
+      },
+      {
+        id: "99b9a82e-8912-4b18-8aa2-bd561ab86d80",
+        description: "Give Alexander's Innards to Jar-Bairn, then reload the area. Where Jar-Bairn was sitting, obtain Companion Jar Talisman."
+      }
+    ]
+  },
+  {
+    id: "2e3ecd46-fd2b-49b9-98a5-a0ee5f02e743",
+    name: "Preceptor Seluvis",
+    url: "https://eldenring.wiki.fextralife.com/Preceptor+Seluvis",
+    requirements: [
+      {
+        id: "d88a5416-f99e-4551-9dfc-71688643547d",
+        description: "Progress Ranni's quest line until you talk to her vassals and Seluvis' Rise is unlocked in the Three Sister's region of Liurnia, north of Caria Manor."
+      },
+      {
+        id: "c5593f73-bb06-4b5c-81d7-9b6b90882b2c",
+        description: "Talk to Seluvis in Seluvis's Rise and receive Seluvis's Potion."
+      },
+      {
+        id: "637f6eca-9f40-45f0-ab58-12458487c9ab",
+        description: "Progress Dung Eater's quest until you can give him Seedbed Curses, but give him Seluvis's Potion. Note: This locks you out of Dung Eater's quest line, and the Blessing of Despair ending until a new NG cycle, and cannot be done the same cycle as feeding it to Nepheli Loux."
```

Comment:
```suggestion
        description: "Progress Dung Eater's quest until you can give him Seedbed Curses, but give him Seluvis's Potion. (This locks you out of Dung Eater's quest line, and the Blessing of Despair ending until a new NG cycle, and cannot be done the same cycle as feeding it to Nepheli Loux.)"
```

I don't really like how long this gets, it makes it hard to read when another line gets added. 
<img width="1540" alt="image" src="https://user-images.githubusercontent.com/14102413/193744358-cae503ef-52a0-4662-8deb-eb672aac07b9.png">
Maybe someday I'll add some note section

Reply:
One thing I looked at doing, but haven't gotten to yet, is adjusting the CSS to left-align the text against the checkbox. Or center align the checkbox with the text vertically. Played with it a bit and decided to just get the quest lines finished out first.